### DISCUSSION TITLE
Weaviate的端口映射修改

### DISCUSSION THREAD

**Discussion by JackHu1120** [langgenius/dify#3202] [General]
我将Weaviate的端口映射修改为42075，请问在web中需要怎么修改，或者是还需要修改其他相关的配置吗，现在向量数据库连接失败
![320478017-a6bdc659-3261-431c-9285-3e4f87878909](https://github.com/langgenius/dify/assets/129717356/f64d9a5a-ce6c-4036-b94d-bc32562b7482)

**Comment by YidaHu** [comment]
.env file in the api directory, change WEAVIATE_ENDPOINT
🫱http://host:port
